server/src/main/java/io/mantisrx/server/master/domain/DataFormatAdapter.java

```diff
@@ -539,12 +548,12 @@ public static MantisJobMetadataWritable convertMantisJobMetadataToMantisJobMetad
     }
 
     public static FilterableMantisJobMetadataWritable convertMantisJobMetadataToFilterableMantisJobMetadataWriteable(IMantisJobMetadata jobMetadata) {
-        long startedAt = (jobMetadata.getStartedAtInstant().isPresent()) ? jobMetadata.getStartedAtInstant().get().toEpochMilli() : -1;
+        Instant startedAtInstant = jobMetadata.getStartedAtInstant().orElse(Instant.ofEpochMilli(0));
```

Comment:
default changed from -1 to 0, expected?

Reply:
Yes I changed the default to 0, as old job definitions when loaded into the new bean by the JSON parser would set the missing startedAt to 0. So just consolidate on one default value

Reply:
gotcha.. minor nit: can use the newly defined constant here instead of hardcoding